pom.xml

```diff
@@ -103,6 +103,13 @@
   </dependencies>
 
   <build>
+    <resources>
```

Comment:
Maybe good to add some comment here as well to clarify this was needed in combination with the pom.properties to be able to access the name & version.